core/tests/webdriverio_utils/BlogDashboardPage.js

```diff
@@ -208,13 +192,17 @@ var BlogDashboardPage = function() {
 
   this.getMatTab = async function(tabName) {
     await waitFor.visibilityOf(
-      matTabLabels.first(), 'Mat Tab Toggle options take too long to appear.');
-    for (i = 0; i < await matTabLabels.count(); i++) {
-      var matTab = matTabLabels.get(i);
+      matTabLabel, 'Mat Tab Toggle options take too long to appear.');
+    var matTabLabels = await matTabLabelsSelector();
+    for (i = 0; i < matTabLabels.length; i++) {
+      var matTab = matTabLabels[i];
       var tabText = await action.getText(
         `Blog Dashboard tab ${i}`, matTab);
       if (tabText.startsWith(tabName)) {
         await action.click(`${tabName} tab`, matTab);
+        await waitFor.visibilityOf(matInkBar);
+        await waitFor.rightTransistionToComplete(
+          matInkBar, `${tabName} tab takes too long to be active`);
```

Comment:
`${tabName} tab takes too long to be active` error message might create confusion among developers while debugging any error that comes because of this as it has nothing to say about the right transistion of the element? Can we not have a more meaningful name?

Reply:
Done.